app/boards/shields/snap/snap_left.conf

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022 The ZMK Contributors
```

Comment:
We should just have one `snap.conf` that contains this, it'll be picked up for left and right builds.